lib/ui/screens/contact_list_screen.dart

```diff
@@ -95,6 +99,15 @@ class _ContactListScreenState extends State<ContactListScreen> {
     );
   }
 
+  _onChanged(String value){
+    setState(() {
```

Comment:
Better if create ValueNotifier<List> _filteredContactListNotifier and use ValueListenableBuilder to rebuild the list.